sovtoken/sovtoken/token_req_handler.py

```diff
@@ -69,7 +69,7 @@ def _validate_xfer_public_txn(request: Request, sum_inputs: int, sum_outputs: in
 
     @staticmethod
     def validate_given_inputs_outputs(inputs_sum, outputs_sum, required_amount, request,
-                                      error_msg_suffix: Optional[str]=None):
+                                      error_msg_suffix: Optional[str] = None):
```

Comment:
I think it is pretty standard in python to not have a space for default values, but it is a little weird with the types.

Reply:
In PEP8 it states that defaults should have spaces in between